modules/ROOT/pages/publish-apis.adoc

```diff
@@ -130,6 +130,8 @@ If no changes are visible, xref:troubleshooting.adoc#refresh-the-cache[refresh t
 
 In the API Community Manager control panel, the *Manage APIs* tab shows a *Search APIs* field.
 
+The tab shows a maximum of 100 results, and shows a message if additional results were omitted. Use the search field to filter the APIs so that no relevant API is omitted.
```

Comment:
```suggestion
The tab shows a maximum of 100 results, followed by a message if additional results were omitted. Use the search field to filter the APIs so that no relevant API is omitted.
```

Reply:
Fixed.